doc/conf.py

```diff
@@ -315,6 +315,7 @@
 		('boards/arm/arduino_due/doc/board', 'boards/arm/arduino_due/doc/index'),
 		('boards/arm/arduino_zero/doc/arduino_zero', 'boards/arm/arduino_zero/doc/index'),
 		('boards/arm/atsamd20_xpro/doc/atsamd20_xpro', 'boards/arm/atsamd20_xpro/doc/index'),
+		('boards/arm/bcm958401m2/doc/board', 'boards/arm/bcm958401m2/doc/index'),
```

Comment:
I don't think this is necessary

Reply:
> I don't think this is necessary

I can remove if you want.  But why are all the other boards listed here?

I would think this is needed for having the board added to the board documentation?  Perhaps it is used to generate an html page like this?
https://docs.zephyrproject.org/latest/boards/index.html

Reply:
I thinks it was for some transition in how we structured docs a while ago.  It shouldn't be needed, please remove.

Reply:
> I thinks it was for some transition in how we structured docs a while ago. It shouldn't be needed, please remove.

Ok thanks - change to doc/conf.py removed.  Sounds like the entire doc/conf.py file should be removed if it is not needed now then??